src/lib.rs

```diff
@@ -26,6 +26,19 @@
 //!
 //! # Use
 //!
+//! ## Logging levels
+//! 
+//! This library makes use of the following macros:
+//! 
+//! - [error](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L73)
+//! - [warn](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L104)
+//! - [info](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L138)
+//! - [debug](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L172)
+//! - [trace](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L209)
+//! 
+//! All of these macros accept formatting strings
+//! similarly to [println!](https://github.com/rust-lang/rust/blob/master/src/libstd/macros.rs#L127)
```

Comment:
Thanks a bunch @jesusbv, and sorry for taking so long to get back to you.

I wonder if you might take a slightly different approach here, and instead of creating a new 'logging levels' section, write some prose that gets this same information across, setting the stage for the detail that comes in following section.

It might begin like

```
# Use

The basic use of the log crate is through the five logging macros: [`error!`], [`warn!`], { etc }, where `error!` represents the highest-priority log level, and `trace!` the lowest. Each of these macros accept format strings similarly to `println!`.
```

For the hyperlinks you'll want to use rustdoc links, not code links; for the internal links, relative links, and the std links absolute links.

So the `error!` link would probably be `./macro.error.html` and the `println!` link `https://doc.rust-lang.org/stable/std/macro.println.html`.

If you convert the list to prose you'll probably want to use out-of-line links like

```
The [`error!`] macro.

[`error!`]: ./macro.error.html
```